### DISCUSSION TITLE
Persistent session data

### DISCUSSION THREAD

**Discussion by decenttp** [ltsp/ltsp#537] [Q&A] [answered]
Hi,

I have setup LTSP server with 1 chrootless image and 1 using VM image. Incase the clients are loading the VM image the files being downloaded and other session data is being lost of reboot. That's not the case with chrootless image though. How can i  fix this?

Thanks

**Comment by alkisg** [comment, marked as answer]
Install the ltsp package inside the VM.

**Reply by decenttp** [reply]
Thanks, that solved it.
